easybuild/easyconfigs/u/UShER/usher-0.4.1-isnan-CMake.patch

```diff
@@ -0,0 +1,43 @@
+Patch to correct CMakeLists.txt so external TBB will be used instead of internal build.
+Patch to convert innan to std::isnan
```

Comment:
Please do these in separate patches. Also, you don't actually need to change the std::isnan for 10.3.0 at least. 
Also the rest of the changes are unncessary. Using find_package works fine. cf. #13708

Reply:
I am happy to split the patch in two, no worries here. 
I cannot use 10.3.0, simply because of the problem I am having with `tbb-2021.3.0` and the missing header file so I am stuck with the older version where we only have the 10.2.0 version of `gcc`.
I am not sure where the std::isnan crept in. I cannot reproduce it any more so I removed that patch, only leaving the patch for CMake. 
I had problems with the CMake patch when I tried that method manually which caused the long discussion thread. 
The patch I came up with works also outside EasyBuild, which I see as an advantage specially when debugging programs. In the end, both work.

Reply:
I have removed the usher-0.4.1-isnan-CMake.patch and replaced it with the usher-0.4.1-CMake.patch one which addresses the CMakeList.txt issue we both noted, and solved differently.